test/e2e/singleton-status/use-kubestellar.sh

```diff
@@ -73,9 +73,9 @@ EOF
 :
 : -------------------------------------------------------------------------
 : "Verify that manifestworks wrapping the objects have been created in the mailbox namespaces"
-: Expect: one header line, one for the nginx namespace, one for the nginx deployment, one for the status agent Deployment
+: Expect: one header line, one for the nginx-singleton-bindingpolicy Binding\'s workload, one for the status agent Deployment
```

Comment:
(A) "workload" is vague here, particularly while issue #1754 is not resolved.

(B) Why should the namespace no longer appear in these listings?